resources/views/layouts/app.blade.php

```diff
@@ -69,6 +69,11 @@ class="navbar-toggler"
                         </li>
                         @endif
                         @else
```

Comment:
Remove?

Reply:
No. Because the Nav link will be destroyed and it is part of  @guest blade directives.